test/unittest/unittest_nntrainer_models.cpp

```diff
@@ -1103,12 +1153,16 @@ INSTANTIATE_TEST_CASE_P(
     mkModelTc(preprocess_flip_validate, "3:1:1:10", 10),
 
     /**< Addition test */
-    mkModelTc(addition_resnet_like_validate, "3:1:1:10", 10)
+    mkModelTc(addition_resnet_like_validate, "3:1:1:10", 10),
 
     /// #1192 time distribution inference bug
     // mkModelTc(fc_softmax_mse_distribute_validate, "3:1:5:3", 1),
     // mkModelTc(fc_softmax_cross_distribute_validate, "3:1:5:3", 1),
     // mkModelTc(fc_sigmoid_cross_distribute_validate, "3:1:5:3", 1)
+    mkModelTc(lstm_basic, "1:1:1:1", 1),
+    mkModelTc(lstm_return_sequence, "1:1:2:1", 1),
+    mkModelTc(lstm_return_sequence_with_batch, "2:1:2:1", 1),
+    mkModelTc(lstm_return_sequence_with_batch_n, "2:1:2:1", 2)
```

Comment:
lets add testcases later with unit size more than 1 and long time sequences to ensure stability of our LSTM implementations.

Reply:
yes. I will later pr